docs/en/writing-running-appium/server-args.md

```diff
@@ -1,23 +1,24 @@
 # Appium server arguments
 
-Since Appium 1.5, many server arguments have been deprecated in favor of the [--default-capabilities flag](/docs/en/writing-running-appium/default-capabilities-arg.md).
+Since Appium 1.5, many server arguments have been removed in favor of the [--default-capabilities param](/docs/en/writing-running-appium/default-capabilities-arg.md).
 
-Usage: `node . [flags]`
+Since Appium 2.0, all driver-specific params have been made accessible via `--driver-args` instead of as their own separate params. See below for details on `--driver-args`.
 
-## Server flags
-All flags are optional, but some are required in conjunction with certain others.
+Usage: `appium [params]` (or `appium server [params]` to be explicit)
 
+## Server params
 
+All params are optional, but some are required in conjunction with certain others.
 
 <expand_table>
 
-|Flag|Default|Description|Example|
+|Param|Default|Description|Example|
 |----|-------|-----------|-------|
-|`--shell`|null|Enter REPL mode||
-|`--allow-cors`|false|Turn on CORS compatibility mode, which will allow connections to the Appium server from within websites hosted on any domain. Be careful when enabling this feature, since there is a potential security risk if you visit a website that uses a cross-domain request to initiate or introspect sessions on your running Appium server.||
 |`-a`, `--address`|0.0.0.0|IP Address to listen on|`--address 0.0.0.0`|
 |`-p`, `--port`|4723|port to listen on|`--port 4723`|
 |`-pa`, `--base-path`|null|Initial path segment where the Appium/WebDriver API will be hosted. Every endpoint will be behind this segment.|`--base-path /wd/hub`|
+|`--driver-args`|`{}`|JSON object string (or path to JSON file) consiting of CLI arguments to send to specific drivers. Should be of the form `{"[driverName]": {"[driverArg1Name]": [driverArg1Value], ...}}`|`--driver-args '{"xcuitest": {"webkitDebugProxyPort": 1234}}'|
```

Comment:
What is the difference between `--dc` and `--driver-args`? 
For example, I now use `--dc` to set uiautomator2ServerLaunchTimeout, but it seems a driver arg.

Reply:
basically driver-args can be anything while default capabilities are, well, only capabilities. Also The latter could be overridden by user-provided values, while driver args have the highest priority and cannot be overridden.

Reply:
Thanks, got it. 👀
The doc seems to need to explain: `the driver args have the highest priority and cannot be overridden.`

Reply:
`--dc` sets default capabilities that will exist on any new session.  drivers might want to get information not from capabilities but from the command line, to ensure that only a system administrator can set it, and not just anyone. in some cases there can also be an overlap: some options can be set both from capabilities or the CLI.

Reply:
oops, didn't realize this conversation already happened when i replied. i think we probably need a special doc explaining the difference between capabilities and arguments, that we can link from here.